python/py-avro/Portfile

```diff
@@ -18,17 +19,36 @@ long_description    ${description}
 
 homepage            https://avro.apache.org/
 
-checksums           rmd160  61c45d1cfa9914306401cb88d880ac63602d43cc \
-                    sha256  f456539183df0d6912daedfbf0f94d7cf69b4fba7200490e8a06d1e8f23af43d \
-                    size    1845163
+checksums           rmd160  e398cc448a227de8754eb60848eb4a47f0633265 \
+                    sha256  4e94b43ebdb1081fd842644bda2caaddc586c1b608efe9236d348d01bf0866a4 \
+                    size    1899682
 
 python.versions     37 38
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                        port:py${python.version}-setuptools
-
-    worksrcdir          ${distname}/lang/py3
```

Comment:
I don't see the benefit of removing this and then needing to add build.dir, destroot.dir, and test.dir - if no obvious reason please revert

Reply:
The original code now puts the *base* in `…/lang/py3`, but then fails because you have to go into another `lang/py3` below that to find `setup.py` to build.

Reply:
that's because you're using `fetch.type git` and I don't think you should

Reply:
The extract fails without it. See above.

> `sudo port extract py37-avro`
```
:info:extract Executing:  cd "/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_python_py-avro/py37-avro/work" && /usr/bin/gzip -dc '/opt/local/var/macports/distfiles/py-avro/avro-1.9.1.tar.gz' | /usr/bin/tar -xf -
:debug:extract system:  cd "/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_python_py-avro/py37-avro/work" && /usr/bin/gzip -dc '/opt/local/var/macports/distfiles/py-avro/avro-1.9.1.tar.gz' | /usr/bin/tar -xf -
:info:extract apache-avro-8921826/lang/java/mapred/src/test/resources/org/apache/avro/mapreduce/mapreduce-test-input.avro/.part-r-00000.avro.crc: Failed to restore metadata
:info:extract tar: Error exit delayed from previous errors.
```

Reply:
see my comment above

Reply:
Moved to pypi.